runtime/compiler/x/i386/codegen/IA32PrivateLinkage.cpp

```diff
@@ -720,3 +720,14 @@ void J9::X86::I386::PrivateLinkage::buildVirtualOrComputedCall(
       }
    }
 
+intptrj_t
+J9::X86::I386::PrivateLinkage::entryPointFromCompiledMethod()
+   {
+   return reinterpret_cast<intptrj_t>(cg()->getCodeStart());
+   }
+
+intptrj_t
+J9::X86::I386::PrivateLinkage::entryPointFromInterpretedMethod()
+   {
+   return reinterpret_cast<intptrj_t>(cg()->getCodeStart());
```

Comment:
Is there any reason to extend this function if the implementation is exactly the same as the base?

Reply:
Um, probably not.  Its been a couple of months since I wrote this so if I did have a reason I don't recall what it is.  My answer then is "no", and I can change the code.

Reply:
@ymanton : it took a while to swap this back into my head.  There was no reason for this and I removed the 386 override.